src/FSharpLint.Core/Rules/Conventions/SourceLength/SourceLengthHelper.fs

```diff
@@ -4,23 +4,90 @@ open System
 open FSharpLint.Framework
 open FSharpLint.Framework.Suggestion
 open FSharp.Compiler.Text
+open FSharpLint.Framework.ExpressionUtilities
+open System.Text.RegularExpressions
 
 [<RequireQualifiedAccess>]
-type Config = { MaxLines:int }
+type Config = 
+    { 
+        MaxLines:int
+        SkipBlankLines: bool
+        SkipComments: bool
+    }
+
+type private MultilineCommentMarker =
+    | Begin of int
+    | End of int
```

Comment:
@webwarrior-ws should be uint?

Reply:
Since indices in .NET are ints, it's logical to use int here too instead of converting numbers back and forth.